CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ option(CMAKE_FIND_PACKAGE_PREFER_CONFIG
 option(ENABLE_SDL2_NET "Enable SDL2_net" On)
 option(ENABLE_SDL2_MIXER "Enable SDL2_mixer" On)
 
-find_package(SDL2 2.0.7)
+find_package(SDL2 2.0.14)
```

Comment:
When was this version released? What are the versions in common LTS distros?

Reply:
It was released in July 2021.

Debian Buster (oldoldstable): 2.0.9
Debian Bullseye (oldstable): 2.0.14
Ubuntu Jammy 20.04LTS: 2.0.10
Ubuntu Focal 22.04LTS: 2.0.20

The alternative here would be to use `SDL_VERSION_ATLEAST` guards in various spots throughout the new gamepad code.